pkgs/by-name/ka/kasmvnc-xvnc/package.nix

```diff
@@ -0,0 +1,125 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  xorgserver,
+  pkg-config,
+  pixman,
+  mesa,
+  autoconf,
+  automake,
+  xorg-autoconf,
+  fontutil,
+  libtool,
+  libX11,
+  libGL,
+  libgcrypt,
+  xtrans,
+  libxkbfile,
+  libXfont2,
+  libXau,
+  libxcvt,
+  libpciaccess,
+  libepoxy,
+}:
+stdenv.mkDerivation rec {
+  pname = "kasmvnc-xvnc";
+  version = "1.3.3";
+  src = fetchFromGitHub {
+    owner = "kasmtech";
+    repo = "KasmVNC";
+    rev = "v${version}";
+    hash = "sha256-1eX6ulDjYjqaZ+VXbu0ogAPpuNHW88uOCfTqKm6Qkso=";
+  };
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    xorg-autoconf
+    fontutil
+    libtool
+    libX11
+    libGL
+    libgcrypt
+    xtrans
+    libxkbfile
+    libXfont2
+    libXau
+    libxcvt
+    libpciaccess
+    libepoxy
+    mesa
+    pkg-config
+    pixman
+  ];
+
+  buildInputs = [
+    #libXfont2
+    #libX11
+    #libXau
+    #libXdmcp
+    #libXext
+    #libXfixes
+    #libXfont2
+    #xorgproto
+    #fontconfig
+    #freetype
+    #libXfont2
+    #xkeyboard_config
+    #libxkbfile
+    #libpciaccess
+    #xorgserver
+  ];
+
+  xorgSource = xorgserver.src;
+
+  patchPhase = ''
+    mkdir -p unix/xserver
+    cp -R ${src}/unix/xserver unix/
+    tar -xf ${xorgSource} -C unix/xserver --strip-components=1
+    cd unix/xserver/
```

Comment:
You cd here and never cd back; that isn't going to work.

Reply:
It is working. It follows their official building guide on xvnc: https://github.com/kasmtech/KasmVNC/blob/master/BUILDING.txt

I do not care about the rest of the source. I only care about the xvnc part, which is located in that folder.